### PR TITLE
Javadoc assertions

### PR DESCRIPTION
**Pull request by jbescos**
Update of JSONP2.0JavadocAssertions.html with new API methods.

### CODE REVIEW

**Comment by jbescos** [comment]
@gurunrao I think you meant JSONP2.1JavadocAssertions.html. I have renamed it to that.

File: tck/tck-docs/assertions/JSONP2.1JavadocAssertions.html [hunk 1]
```diff
@@ -2654,6 +2654,24 @@ <h2>  Jakarta JSON Processing - 2.0<br>
...
+</font></TD><TD align="center" valign="center"><font size="1PT">true</font></TD>
+</TR>
+<TR>
+<TD align="center" valign="center"><a name="JSONP:JAVADOC:682"></a><font size="1PT">JSONP:JAVADOC:682</font></TD><TD align="center" valign="center"><font size="1PT">JsonNumber</font></TD><TD align="left" valign="center"><font size="1PT">jakarta.json.Json.createValue(Number value)<br>
+</font></TD><TD align="left" valign="center"><font size="1PT">Creates a JsonNumber.</font></TD><TD align="center" valign="center"><font size="1PT">true</font></TD><TD align="center" valign="center"><font size="1PT">
+<br>
+</font></TD><TD align="center" valign="center"><font size="1PT">true</font></TD>
+</TR>
+<TR>
+<TD align="center" valign="center"><a name="JSONP:JAVADOC:683"></a><font size="1PT">JSONP:JAVADOC:682</font></TD><TD align="center" valign="center"><font size="1PT">Event</font></TD><TD align="left" valign="center"><font size="1PT">jakarta.json.stream.JsonParser.currentEvent<br>
```
Comment (alwin-joseph) [inline tck/tck-docs/assertions/JSONP2.1JavadocAssertions.html:2670]: typo here JSONP:JAVADOC:682 should be JSONP:JAVADOC:683 ?

**Comment by jbescos** [comment]
There are many unrelated files failing now because of the copyright 2022:
```
/home/runner/work/jsonp/jsonp/api/src/main/javadoc/overview.html: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/module-info.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonValue.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonMergePatch.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonReader.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonReaderFactory.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonArrayBuilder.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonPointer.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/package-info.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonString.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonObject.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonWriterFactory.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonValueImpl.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/stream/JsonGeneratorFactory.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/stream/JsonLocation.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/stream/JsonGenerator.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/stream/package-info.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/stream/JsonGenerationException.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/stream/JsonParser.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/stream/JsonParsingException.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/stream/JsonCollectors.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/stream/JsonParserFactory.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonWriter.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonArray.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/EmptyObject.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonObjectBuilder.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/EmptyArray.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonPatch.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonConfig.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/Json.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonStructure.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonBuilderFactory.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonPatchBuilder.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonNumber.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/spi/package-info.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/spi/JsonProvider.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/src/main/java/jakarta/json/JsonException.java: Copyright year is wrong; is 2021, should be 2022
/home/runner/work/jsonp/jsonp/api/pom.xml: Copyright year is wrong; is 2021, should be 2022
```

**Comment by lukasj** [comment]
did you try to set [a property](https://github.com/eclipse-ee4j/jsonp/blob/master/api/pom.xml#L80) to `true`?

**Comment by jbescos** [comment]
> did you try to set [a property](https://github.com/eclipse-ee4j/jsonp/blob/master/api/pom.xml#L80) to `true`?

I didn't. I have done it now.

**Review by lukasj — COMMENTED** [review]
update the year at least in files you have updated, please

**Comment by jbescos** [comment]
> update the year at least in files you have updated, please

I think that has to be done in the files that were modified in commits of 2022:
- https://github.com/eclipse-ee4j/jsonp/pull/348/commits/06497fdabe54280cb5d0b3ac353bbcee29ed172a
- https://github.com/eclipse-ee4j/jsonp/pull/348/commits/188c86323d42cc1aa20e0694e09a58c6492aa042

I can update all the files to 2022 and it will be correct too, because I will create a new commit in 2022 for that.

**Review by lukasj — APPROVED** [review]
LGTM (squash creates new commit in this year...)

**Pull request merged**